qualtran/bloqs/factoring/mod_sub.py

```diff
@@ -94,6 +97,15 @@ def build_composite_bloq(self, bb: 'BloqBuilder', x: Soquet, y: Soquet) -> Dict[
     def pretty_name(self) -> str:
         return f'y = y - x mod {self.p}'
 
+    def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
+        # Ignore allocation, deallocation, splits and joins for now.
```

Comment:
don't necessarily need to note this.

can you add a qlt_testing.assert_equilvalent_bloq_counts?

Reply:
done